edn/src/types.rs

```diff
@@ -143,6 +162,16 @@ impl Value {
     def_as!(as_list, List, LinkedList<Value>);
     def_as!(as_set, Set, BTreeSet<Value>);
     def_as!(as_map, Map, BTreeMap<Value, Value>);
+
+    pub fn from_bigint(src: &str) -> Result<Value, NewValueError> {
```

Comment:
This might be simpler if we return `Option<Value>`.  Might call it `edn::Value::bigint_from(src: Into<str>)` as well.